rkt/api_service.go

```diff
@@ -636,8 +639,54 @@ func (s *v1AlphaAPIServer) InspectImage(ctx context.Context, request *v1alpha.In
 	return &v1alpha.InspectImageResponse{Image: image}, nil
 }
 
+type LogsStreamWriter struct {
+	server v1alpha.PublicAPI_GetLogsServer
+}
+
+func (sw LogsStreamWriter) Write(b []byte) (int, error) {
+	if err := sw.server.SendMsg(b); err != nil {
+		return 0, err
+	}
+	return len(b), nil
+}
+
 func (s *v1AlphaAPIServer) GetLogs(request *v1alpha.GetLogsRequest, server v1alpha.PublicAPI_GetLogsServer) error {
-	return fmt.Errorf("not implemented yet")
+	path := fmt.Sprintf("/var/lib/rkt/pods/run/%s/stage1/rootfs/var/log/journal/", request.PodId)
```

Comment:
- Instead of `/var/lib/rkt`, use [getDataDir()](https://github.com/coreos/rkt/blob/master/rkt/rkt.go#L254)
- `stage1/rootfs` only works with `--no-overlay`. With overlayfs, use `overlay/deps-sha512-$UUID/upper/` instead. See [getStatusDir()](https://github.com/coreos/rkt/blob/master/rkt/pods.go#L1033) for the logic.
- Is there a way to validate that `request.PodId` is a valid pod id before opening the journal? To be more resistant to unfriendly calls with "../../../" or similar.